### PR TITLE
Add the spamton command.

### PR DESCRIPTION
**Pull request by GenericProgrammer1234**
URGENT PR, GREATEST PR OF THE 21ST CENTURY, HIGH PRIORITY MERGE FOR MORE [[Hyperlink Blocked]].

(I'll actually implement something useful soon, this is just me learning how to add new commands to the bot)

### CODE REVIEW

File: vyxalbot2/chat/__init__.py [hunk 1]
```diff
@@ -500,3 +500,7 @@ async def commitCommand(self, event: EventInfo):
             yield "Failed to get commit info!"
         else:
             yield f"Commit: {result.stdout.decode('utf-8').strip()}"
+
+    async def spamtonCommand(self, event: eventInfo):
```
Comment (GingerIndustries) [inline vyxalbot2/chat/__init__.py:504]: `EventInfo` needs to be capitalized

File: vyxalbot2/chat/__init__.py [hunk 2]
```diff
@@ -500,3 +500,7 @@ async def commitCommand(self, event: EventInfo):
...
+        """All I know is that it is related to spam, PLEASE DON'T DO ANYTHING TO ME, ALL I KNOW IS THIS, I PROMISE!"""
```
Comment (GingerIndustries) [inline vyxalbot2/chat/__init__.py:505]: Probably best to remove the help for this

File: vyxalbot2/chat/__init__.py [hunk 3]
```diff
@@ -500,3 +500,7 @@ async def commitCommand(self, event: EventInfo):
...
+        yield random.choice(["IT'S EV3RY BUDDY 'S FAVORITE [[Number 1 Rated SalesmanChatbot1997]]", "YOU WANT [[Hyperlink Blocked]], DON'T YOU.", "KID!!! IN BUSINESS YOU NEED TO SAY YOU'RE INTERESTED!!!", "MEET LOCAL SINGLES STRAIGHT FROM [My]", "IF YOU'VE [[Lost Control Of Your Life]] THEN YOU JUST GOTTA GRAB IT BY THE [[Silly Strings]]", "DON'T FORGET TO [Star This Message] FOR MORE [Hyperlink Blocked]!"])
```
Comment (GingerIndustries) [inline vyxalbot2/chat/__init__.py:506]: These messages should go in `messages.toml`

**Review by GingerIndustries — CHANGES_REQUESTED** [review]

**Comment by GenericProgrammer1234** [comment]
Right, right, we don't want the FBI to open up on our doors.

**Comment by GenericProgrammer1234** [comment]
How do I use the messages in `messages.toml`?

**Comment by GenericProgrammer1234** [comment]
@GingerIndustries?

**Comment by GenericProgrammer1234** [comment]
Okay wait I figured it out myself.

**Comment by GenericProgrammer1234** [comment]
@GingerIndustries By the way I am The_AH if you were wondering.

**Comment by GingerIndustries** [comment]
[Rejected by lyxal](https://chat.stackexchange.com/transcript/message/64719694#64719694)

**Pull request closed without merging**